script.js

```diff
@@ -0,0 +1,7 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
For code like this that's not super readable, let's add a short comment about what this function does